tests/ossfuzz/config/fuzz_decode_raw_lzma2.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
+dict = fuzz_xz.dict
```

Comment:
Raw LZMA2 does not use the .xz magic bytes, so this dictionary isn't that helpful for the fuzz test